lib/rust/parser/src/syntax/tree.rs

```diff
@@ -68,13 +68,17 @@ macro_rules! with_ast_definition { ($f:ident ($($args:tt)*)) => { $f! { $($args)
             pub error: Error,
             pub ast: Tree<'s>,
         },
-        Module {
+        Block {
             pub statements: Vec<Tree<'s>>,
         },
         /// A simple identifier, like `foo` or `bar`.
         Ident {
             pub token: token::Ident<'s>,
         },
+        /// A numeric constant, like `10`.
```

Comment:
I'd call it "A numeric literal", just to use the same nomenclature across files.